cmd/admin-bucket-quota.go

```diff
@@ -103,7 +103,7 @@ EXAMPLES:
 	  {{.Prompt}} {{.HelpName}} myminio/mybucket --fifo 64kB
```

Comment:
isn't this also inconsistent?

Reply:
this is the convention to write kilo bytes afaics.

Reply:
We should support both styles @poornas also do we really care about kilobytes fifo? Are we trying to show that you can only store 1 file? 

Examples can be a bit more geared towards practical usecases.